src/storage/v2/transaction.hpp

```diff
@@ -36,6 +36,7 @@ struct Transaction {
         start_timestamp(start_timestamp),
         command_id(0),
         must_abort(false),
+        aborted_by_user(false),
```

Comment:
This variable is bool type, so you can change it to `is_aborted_by_user`.

Reply:
Check for example in transaction.hpp naming for is_transaction_active_.

Reply:
yes agree! Can you check constructor `Transctions &&` ? I am not sure if this line:
```
is_aborted_by_user_(false),
```
can create some problems and:
```
is_aborted_by_user_(other.is_aborted_by_user_)
```
cannot be written because this constructor is deleted from atomic